docs/features/blocks/layoutblocks/column.rst

```diff
@@ -0,0 +1,21 @@
+.. _column-block:
+
+Column Block
+============
+
+Creates columns in the layout
+
+Field Reference
+---------------
+
+Fields and purposes:
+
+* **Column Size** - Sets the size of the columns
+
+* **Content** - Options for the content blocks to display in the columns
```

Comment:
Can you also indicate some of the advanced settings fields... I think there is one that controls the breakpoint (corresponding to bootstrap sm, md, etc.)

Reply:
done. also linked to bootstrap docs